semcore/data-table/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.29.0] - 2024-03-12
+
+### Added
+
+- Prop `withScrollBar` for enable scroll in `DataTable.Head`.
```

Comment:
`for` -> `to`
`scroll` -> `scrolling`

(Sorry for grammar attention)

Reply:
fixed